src/history.js

```diff
@@ -187,14 +187,14 @@ class History extends Emitter {
     if (this.size <= 0) {
       this.begin()
       return
-    } else if (!next) {
+    } else if (action === this.head) {
       next = this.head = parent
     } else if (action === this.root) {
       this.root = next
     }
 
     if (!action.disabled) {
-      this.reconcile(next)
+      this.reconcile(this.head)
```

Comment:
ignore last question there, understanding how a microcosm reacts to the `update` emission from history
```js
let next = this.recall(action.parent, this.initial)
```